### ISSUE TITLE
Can't pip install --no-binary :all: for rsa 4.1

### ISSUE THREAD

**Issue by daveyeager** [sybrenstuvel/python-rsa#148]
Get the following error attempting to pip install --no-binary :all: for rsa 4.1:

```
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python /usr/local/lib/python3.7/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-t_dd66v5/overlay --no-warn-script-location --no-binary :all: --only-binary :none: -i https://pypi.org/simple -- 'poetry>=0.12'
       cwd: None
  Complete output (49 lines):
  Looking in indexes: https://pypi.org/simple
  Collecting poetry>=0.12
    Downloading poetry-1.0.9.tar.gz (167 kB)
    Installing build dependencies: started
    Installing build dependencies: finished with status 'done'
    Getting requirements to build wheel: started
    Getting requirements to build wheel: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python /usr/local/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpj1ktbbkf
         cwd: /tmp/pip-install-ap8po1db/poetry
    Complete output (36 lines):
    Traceback (most recent call last):
      File "/usr/local/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
        main()
      File "/usr/local/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "/usr/local/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
        return hook(config_settings)
      File "/tmp/pip-build-env-np3nbcwo/overlay/lib/python3.7/site-packages/intreehooks.py", line 53, in get_requires_for_build_wheel
        return self._backend.get_requires_for_build_sdist(config_settings)
      File "/tmp/pip-build-env-np3nbcwo/overlay/lib/python3.7/site-packages/intreehooks.py", line 38, in _backend
        obj = self._module_from_dir(modname)
      File "/tmp/pip-build-env-np3nbcwo/overlay/lib/python3.7/site-packages/intreehooks.py", line 25, in _module_from_dir
        mod = importlib.import_module(modname)
      File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 728, in exec_module
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "/tmp/pip-install-ap8po1db/poetry/poetry/masonry/__init__.py", line 10, in <module>
        from .builder import Builder
      File "/tmp/pip-install-ap8po1db/poetry/poetry/masonry/builder.py", line 1, in <module>
        from .builders.complete import CompleteBuilder
      File "/tmp/pip-install-ap8po1db/poetry/poetry/masonry/builders/__init__.py", line 1, in <module>
        from .complete import CompleteBuilder
      File "/tmp/pip-install-ap8po1db/poetry/poetry/masonry/builders/complete.py", line 6, in <module>
        from poetry.factory import Factory
      File "/tmp/pip-install-ap8po1db/poetry/poetry/factory.py", line 10, in <module>
        from clikit.api.io.io import IO
    ModuleNotFoundError: No module named 'clikit'
```

**Comment by lhoestq** [comment]
Same issue here

**Comment by sybrenstuvel** [comment]
I can reproduce the issue. The nasty thing is that even `pip install --no-binary :all: clikit` fails on `ModuleNotFoundError: No module named 'clikit'`. Apparently this is a known issue with Poetry: https://github.com/python-poetry/poetry/issues/1975

Python-RSA should always be installable in a source/Python-only environment. If there is no workaround, I'm afraid I'll have to go back to using bare `requirements.txt` and `setup.py` files again :'(

**Comment by eddie-scio** [comment]
Just wanted to say thanks Sybren for the quick fix and publishing `4.1.1.` to pypi!

**Comment by sybrenstuvel** [comment]
I just released 4.2 that fixes this and hopefully also #147 :)
